scripts/bundle-esm.js

```diff
@@ -1,27 +1,26 @@
 'use strict';
 
-/* eslint-disable no-console */
-/* eslint-disable guard-for-in */
-/* eslint-disable node/no-extraneous-require */
+/* eslint-disable node/no-extraneous-require -- these are all installed as part of `parcel` */
 
 const bundleReport = require('@parcel/reporter-cli/lib/bundleReport').default;
 const defaultConfigContents = require('@parcel/config-default');
+const defaultConfigFilePath = require.resolve('@parcel/config-default');
```

Comment:
Just interesting, why parcel? rollup and webpack, even esbuild is better, parcel is very buggy

Reply:
At the time I wrote this, Parcel looked like the simplest way to take CommonJS and output ESM. I didn't put much more thought into it than picking 'any' bundler to use for a proof of concept.